Formula/weather.rb

```diff
@@ -0,0 +1,26 @@
+class Weather < Formula
+  desc "Command-line weather"
+  homepage "https://github.com/jessfraz/weather"
+  url "https://github.com/jessfraz/weather/archive/v0.10.0.tar.gz"
+  sha256 "e6da8b23548f6ed168089e5ca946c96fdacb252b79695a79993954e63faf2bd5"
+
```

Comment:
maybe add a `head` with `head "https://github.com/uNetworking/uWebSockets.git"`?

Reply:
let's skip this for now.